test/build-test/prepublish/lib/build-spec.js

```diff
@@ -0,0 +1,37 @@
+const sinon = require( 'sinon' );
+const chai = require( 'chai' );
+const expect = chai.expect;
+const sinonChai = require( 'sinon-chai' );
+chai.use( sinonChai );
```

Comment:
Prob could move this below the imports, so the imports happen and then they are used.